pool/db.go

```diff
@@ -225,13 +227,11 @@ func InitDB(dbFile string, isSoloPool bool) (*bolt.DB, error) {
 
 	if switchMode {
 		// Backup the current database and wipe it.
-		now := time.Now().Format(time.RFC3339)
-		file := fmt.Sprintf("dcrpool@%v.kv", now)
-		err := backup(db, file)
+		err := backup(db, backupFile)
 		if err != nil {
 			return nil, err
 		}
-		log.Infof("Pool mode changed, database backup %s created.", file)
+		log.Infof("Pool mode changed, database backup created.")
```

Comment:
I really think we should log the path and filename where the backup was created. Perhaps we could add it to `backup()` in `db.go` to ensure it is always logged and to prevent duplication?

Reply:
Not logging the path of the backup because the backup file location will be consistent, it'll always be created where the db resides. Probably should state this in th README as well.

Reply:
Understood.